src/app/calendar/getParcelCalendarEvents.ts

```diff
@@ -0,0 +1,83 @@
+import supabase, { Schema } from "@/supabase";
+import { CalendarEvent } from "@/components/Calendar/Calendar";
+
+const COLLECTION_DURATION_MS = 30 * 60 * 1000;
+
+const colorMap: { [location: string]: string } = {
+    "Brixton Hill - Methodist Church": "#d50300",
```

Comment:
as discussed in person, try and 'recreate' the colours you want with a colour picker or with a free palette picker online (make sure you have the rights for it and don't just nick exact colours!)

Reply:
Colours changed - no longer being nicked from any large multinational conglomerates